mne/decoding/search_light.py

```diff
@@ -338,7 +358,10 @@ def _sl_score(estimators, X, y):
     """
     n_iter = X.shape[-1]
     for ii, est in enumerate(estimators):
-        _score = est.score(X[..., ii], y)
+        if scoring is not None:
+            _score = scoring(est, X[..., ii], y)
```

Comment:
A sklearn score returns a float per fold. Look maybe at multi_output scorers






On Thu, Aug 11, 2016 at 3:32 PM +0200, "Jean-Rémi KING" <notifications@github.com> wrote:












In mne/decoding/search_light.py:
> @@ -338,7 +358,10 @@ def _sl_score(estimators, X, y):
>      """
>      n_iter = X.shape[-1]
>      for ii, est in enumerate(estimators):
> -        _score = est.score(X[..., ii], y)
> +        if scoring is not None:
> +            _score = scoring(est, X[..., ii], y)



@agramfort this is a single/multi class scoring problem. Isn't there a way to handle thus by sklearn scorers?



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.